lib/adapters/REST/endpoints/asset.ts

```diff
@@ -16,13 +16,16 @@ import { normalizeSelect } from './utils'
 
 export const get: RestEndpoint<'Asset', 'get'> = (
   http: AxiosInstance,
-  params: GetSpaceEnvironmentParams & { assetId: string } & QueryParams
+  params: GetSpaceEnvironmentParams & { assetId: string } & QueryParams,
+  _rawData?: never,
```

Comment:
```suggestion
  _rawData?: unknown,
```

Reply:
Shouldn't it become `rawData?: unknown` (without `_`) if we're not using `never`?

Reply:
I updated it with `rawData?: unknown`, linter might complain at some point for the unused var